### DISCUSSION TITLE
.NET MAUI 8.0.71 (Service Release 7.1)

### DISCUSSION THREAD

**Discussion by davidortinau** [dotnet/maui#23955] [Announcements]
This release is now available via [NuGet.org](https://www.nuget.org/packages/Microsoft.Maui.Controls/8.0.71) and is compatible with the latest .NET 8 workload. 

```
<ItemGroup>
  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.71" />
  <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.71" />
</ItemGroup>
```

The focus of our service releases is resolving high-impact bugs for developers upgrading their applications from Xamarin to .NET 8.

- [Release notes](https://github.com/dotnet/maui/releases/tag/8.0.71)

**Comment by maonaoda** [comment]
I can't believe this is an update after 23 days◝₍ᴑ̑ДO͝₎◞

**Reply by MartyIX** [reply]
I suppose this is just a minor SR is to unblock some users.

Many PRs were merged to the `main` branch which will turn into .NET SR 8. So it's not like nothing was done in the period.

**Reply by HuaFangYun** [reply]
SR 8 will be released next week ？

**Reply by bronteq** [reply]
Usually major releases are published the 2nd tuesday of the month

**Comment by zuo99** [comment]
We are encountering a major issue with MAUI 8 apps. When the APK is handed over to the publisher, the signing process disrupts the alignment. During installation (signature verification) and running the app, we encounter an offset error with the assemblies/assemblies.blob file in the APK.

Since it's impractical to influence the publisher's release process and realign the APK each time, we need a solution that allows the app to run normally without re-aligning and re-signing the APK.
